vagrant-spk

```diff
@@ -162,15 +162,18 @@ GITIGNORE_CONTENTS = r"""
 GLOBAL_SETUP_SCRIPT = r"""#!/bin/bash
 set -euo pipefail
 
+# Set options for curl. Since we only want to show errors from these curl
+# commands, we also use 'cat' to buffer the output; for more information:
+# https://github.com/sandstorm-io/vagrant-spk/issues/158
 CURL_OPTS="--silent --show-error"
 echo localhost > /etc/hostname
 hostname localhost
-curl $CURL_OPTS https://install.sandstorm.io/ > /host-dot-sandstorm/caches/install.sh
+curl $CURL_OPTS https://install.sandstorm.io/ 2>&1 > /host-dot-sandstorm/caches/install.sh | cat
```

Comment:
Oh, I see that's the proof exists to show that this is in fact okay.

I guess I'd appreciate a comment on the fact that it's a very tricksy line.